misc/vasp/python/processors/Chgcar.py

```diff
@@ -0,0 +1,197 @@
+# Name: Chgcar
+
+import inviwopy as ivw
+import atomdata
+
+import functools
+import math
+import numpy
+
+
+def parseFile(file):
```

Comment:
Probably want to document what types of files that are supported (chgar?) and what it returns. E.g. what are pos, elem and elemtype? Volume will be single channel?
Could also add a link further describing CHGCAR ( https://cms.mpi.univie.ac.at/wiki/index.php/CHGCAR ?)

Reply:
If you know what VASP is then you will know what chgcar means....